### PR TITLE
567 - Fix FeatureFlag logic

### PR DESCRIPTION
**Pull request by BradHN1**
Make the feature flags work for EVERYONE, ALL_EXCEPT and SPECIFIED communities and users.

### CODE REVIEW

**Review by abdullai-t — APPROVED** [review]
LGTM ✅

File: src/database/models.py [hunk 1]
```diff
@@ -442,9 +442,16 @@ def full_json(self):
                 l = loc.zipcode
             locations += l
 
-
-        features_flags = FeatureFlag.objects.filter(communities=self)
-        features_flags_json = [f.simple_json() for f in features_flags]
+        # Feature flags can either enable features for specific communities, or disable them
+        feature_flags = FeatureFlag.objects.all()
```
Comment (Opoku-Agyemang) [inline src/database/models.py:446]: why are we fetching all feature flags here?  I really dont think this logic should live here.  We have a separate endpoint to fetch feature flags based on some specific filters.  Why not use that?

**Review by Opoku-Agyemang — CHANGES_REQUESTED** [review]

**Comment by BradHN1** [comment]
There will always be a very small number of feature flags.  And we need ti to work this week, instead of being completely broken.

File: src/database/models.py (hunk 1 above)
Comment (BradHN1) [reply src/database/models.py:446]: We need this working now, and it was broken before., and the time penalty will be very small for loading a community.  There are many much slower things to worry about.

**Pull request merged**